packages/terra-time-input/CHANGELOG.md

```diff
@@ -3,8 +3,13 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Removed Ignore Accessibility condition from wdio tests.
+* Changed help text color to match required contrast ratio.
+
 ### Added
 * Added documentation on required peerDependencies
+
 ### Added
```

Comment:
It would be nice if these two Added sections were combined

Reply:
Resolved in 009e0a3